addons/mrp/models/mrp_production.py

```diff
@@ -2884,6 +2884,21 @@ def _autoprint_generated_lot(self, lot_id):
             clean_action(action, self.env)
             return action
 
+    def _autoprint_mass_generated_lots(self):
+        actions = []
+        productions_to_print = self.filtered(lambda p: p.picking_type_id.auto_print_generated_mrp_lot)
+        productions_by_print_formats = productions_to_print.grouped(lambda p: p.picking_type_id.generated_mrp_lot_label_to_print)
```

Comment:
Note: Looks like the mass produce feature doesn't work correctly for more than 1 MO at a time :sweat_smile: but it doesn't hurt to keep it like this in case someone needs to fix it later on in the future.